execution/contexts/identify_context.go

```diff
@@ -0,0 +1,61 @@
+package contexts
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/burrow/acm/validator"
+
+	"github.com/hyperledger/burrow/acm/acmstate"
+	"github.com/hyperledger/burrow/execution/errors"
+	"github.com/hyperledger/burrow/execution/exec"
+	"github.com/hyperledger/burrow/execution/registry"
+	"github.com/hyperledger/burrow/logging"
+	"github.com/hyperledger/burrow/txs/payload"
+)
+
+type IdentifyContext struct {
+	NodeWriter   registry.ReaderWriter
+	StateReader  acmstate.Reader
+	ValidatorSet validator.Reader
+	Logger       *logging.Logger
+	tx           *payload.IdentifyTx
+}
+
+func (ctx *IdentifyContext) Execute(txe *exec.TxExecution, p payload.Payload) error {
+	var ok bool
+	ctx.tx, ok = p.(*payload.IdentifyTx)
+	if !ok {
+		return fmt.Errorf("payload must be IdentifyTx, but is: %v", txe.Envelope.Tx.Payload)
+	}
+
+	// validators are tightly bound to input accounts
```

Comment:
I don't really know what you mean here and I have most of the context. Can you make more plain and explicit?

It looks like currently all we are saying is - `if you have bond permission, then you can submit identify txs`. I'm fine with this.

So I think the comment ought to be something like `we only allow those who could become validators to identify nodes, therefore expect the input account to have bond permission`

Reply:
Okay it's only on closer inspection I realise that the input address is being bound as the validator address associated with this node key - this needs to be much more clearly documented. Possibly by including it explicitly in RegisteredNode